tests/testthat/test-experimentSpec.R

```diff
@@ -144,26 +144,26 @@ test_that("experimentSpec module works as expected in the test app", {
 
   # Filtering out all samples does give a validation message, so we are safe
   # downstream.
-  app$set_inputs(!!ns2("add_MAE_filter-subjects-var_to_add") := "SEX")
+  app$set_inputs(!!ns2("add-MAE-subjects-var_to_add") := "SEX")
   app$wait_for_idle()
-  app$set_inputs(!!ns2("MAE_filter-subjects-_var_SEX-content-inputs-selection") := character())
+  app$set_inputs(!!ns2("active-MAE-subjects-MAE_SEX-inputs-selection") := character())
   app$wait_for_idle()
 
   res <- app$get_value(output = ns("summary"))
   expect_match(res$message, "No genes or samples included in this experiment, please adjust filters")
-  app$click(ns2("MAE_filter-remove_filters"))
+  app$click(ns2("active-MAE-remove_filters"))
 
   # Same for filtering out all genes.
-  app$set_inputs(!!ns2("add_MAE_filter-hd2-row_to_add") := "chromosome")
+  app$set_inputs(!!ns2("add-MAE-hd2-row_to_add") := "chromosome")
   app$wait_for_idle()
-  app$set_inputs(!!ns2("MAE_filter-hd2-rowData_var_chromosome-content-inputs-selection") := character())
+  app$set_inputs(!!ns2("active-MAE-hd2-MAE_chromosome_hd2_subset-inputs-selection") := character())
```

Comment:
@danielinteractive @BFalquet 
Since remote browser can't change these values, with your permission guys, I'd like to remove these two tests.

Reply:
@gogonzo yes, just curious why this is not testable?

Reply:
@danielinteractive don't know why, I've tried to debug it but no success. This is why I suggest to remove it and test the app manually instead to make sure that everything works (from my perspective everything is fine)

Reply:
just discussed with @BFalquet , and we can take it out for now here. I will file follow up issue so we can drill down on this, because it seems like a bug somewhere

Reply:
in https://github.com/insightsengineering/teal.modules.hermes/issues/315

Reply:
@gogonzo @danielinteractive  found a fix (but i dont understand the mechanism behind):
```
  app$set_inputs(!!ns2("active-MAE-hd2-MAE_chromosome_hd2_subset-inputs-selection_open") := TRUE, allow_no_input_binding_ = TRUE)
  app$set_inputs(!!ns2("active-MAE-hd2-MAE_chromosome_hd2_subset-inputs-selection") := character(0))
  app$set_inputs(!!ns2("active-MAE-hd2-MAE_chromosome_hd2_subset-inputs-selection_open") := FALSE, allow_no_input_binding_ = TRUE)
```

Reply:
Great! I'll have a look at this tomorrow. Looks legit ;)

Reply:
Thanks @BFalquet looks like your proposition fixes the problem! Thanks